harc_game_web/apps/core/templates/core/navbar.html

```diff
@@ -14,11 +14,10 @@
 
   <a href="{% url 'word_of_the_day' %}"><button class="rpgui-button" type="button">Hasło dnia</button></a>
   <a href="{% url 'all_teams' %}"><button class="rpgui-button" type="button">Drużyny</button></a>
-  <a href="{% url 'all_posts' %}"><button class="rpgui-button" type="button">Posty</button></a>
+  <!--a href="{% url 'all_posts' %}"><button class="rpgui-button" type="button">Posty</button></a-->
```

Comment:
to tak forever czy póki co?

Reply:
Przeniosłem na stronę główną, ale nie wiem czy ma ten button dodatkowo być czy nie...